extism.go

```diff
@@ -365,6 +365,10 @@ func NewPlugin(
 	// See: https://github.com/extism/go-sdk/pull/1#issuecomment-1650527495
 	moduleConfig = moduleConfig.WithStartFunctions().WithFSConfig(fs)
 
+	if c.hasWasi && os.Getenv("EXTISM_ENABLE_WASI_OUTPUT") == "1" {
```

Comment:
Maybe something like this, which will work when `EXTISM_ENABLE_WASI_OUTPUT` is set to any value:
```suggestion
   _, wasiOutput := os.LookupEnv("EXTISM_ENABLE_WASI_OUTPUT")
	if c.hasWasi && wasiOutput {
```

 I think this is how it works in Rust too.

Reply:
good call!